### PR TITLE
Revert breaking docs changes and add disclaimers for new features

### PR DESCRIPTION
**Pull request by gewenyu99**
…il 1.5 on Cloud

## What does this PR do?

Some features aren't available yet on Cloud. Add disclaimers and remove breaking changes so people stop hitting head against wall when using Cloud.
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 1]
```diff
@@ -105,36 +105,36 @@ interface Todo {
...
 
     await databases.createStringAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'title',
-        255,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'title', 
+        255,
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:122]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'title',
        255,
```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 2]
```diff
@@ -105,36 +105,36 @@ interface Todo {
...
 
     await databases.createStringAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'description',
-        255, false,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'description', 
+        255, false,
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:130]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'description',
        255, false,
```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 3]
```diff
@@ -105,36 +105,36 @@ interface Todo {
...
-        'description',
-        255, false,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'description', 
+        255, false, 
         'This is a test description'
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:133]:
```suggestion

```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 4]
```diff
@@ -105,36 +105,36 @@ interface Todo {
...
     );
-
+    
     await databases.createBooleanAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'isComplete',
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'isComplete',
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:137]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'isComplete',
```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 5]
```diff
@@ -164,23 +164,23 @@ async function seedDatabase(): Promise<void> {
     };
 
     await databases.createDocument(
-        todoDatabase.$id,
-        todoCollection.$id,
-        ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        ID.unique(),
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:169]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        ID.unique(),
```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 6]
```diff
@@ -164,23 +164,23 @@ async function seedDatabase(): Promise<void> {
...
-        todoDatabase.$id,
-        todoCollection.$id,
-        ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        ID.unique(), 
         testTodo1
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:172]:
```suggestion

```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 7]
```diff
@@ -164,23 +164,23 @@ async function seedDatabase(): Promise<void> {
...
     );
-
+    
     await databases.createDocument(
-        todoDatabase.$id,
-        todoCollection.$id,
-        ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        ID.unique(),
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:183]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        ID.unique(),
```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 8]
```diff
@@ -205,7 +205,7 @@ async function getTodos(): Promise<void> {
 }
 
 async function runAllTasks(): Promise<void> {
-    await prepareDatabase();
+    await prepareDatabase();
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:208]:
```suggestion
    await prepareDatabase();
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 9]
```diff
@@ -94,36 +94,36 @@ var todoCollection;
...
 
     await databases.createStringAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'title',
-        255,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'title', 
+        255,
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:111]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'title',
        255,
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 10]
```diff
@@ -94,36 +94,36 @@ var todoCollection;
...
 
     await databases.createStringAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'description',
-        255, false,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'description', 
+        255, false,
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:119]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'description',
        255, false,
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 11]
```diff
@@ -94,36 +94,36 @@ var todoCollection;
...
-        'description',
-        255, false,
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'description', 
+        255, false, 
         'This is a test description'
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:122]:
```suggestion

```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 12]
```diff
@@ -94,36 +94,36 @@ var todoCollection;
...
     );
-
+    
     await databases.createBooleanAttribute(
-        todoDatabase.$id,
-        todoCollection.$id,
-        'isComplete',
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        'isComplete',
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:126]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        'isComplete',
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 13]
```diff
@@ -153,23 +153,23 @@ async function seedDatabase() {
     };
 
     await databases.createDocument(
-        todoDatabase.$id,
-        todoCollection.$id,
-        sdk.ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        sdk.ID.unique(),
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:158]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        sdk.ID.unique(),
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 14]
```diff
@@ -153,23 +153,23 @@ async function seedDatabase() {
...
-        todoDatabase.$id,
-        todoCollection.$id,
-        sdk.ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        sdk.ID.unique(), 
         testTodo1
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:161]:
```suggestion

```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 15]
```diff
@@ -153,23 +153,23 @@ async function seedDatabase() {
...
     );
-
+    
     await databases.createDocument(
-        todoDatabase.$id,
-        todoCollection.$id,
-        sdk.ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        sdk.ID.unique(),
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:165]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        sdk.ID.unique(),
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 16]
```diff
@@ -153,23 +153,23 @@ async function seedDatabase() {
...
-        todoDatabase.$id,
-        todoCollection.$id,
-        sdk.ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        sdk.ID.unique(), 
         testTodo2
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:168]:
```suggestion

```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 17]
```diff
@@ -153,23 +153,23 @@ async function seedDatabase() {
...
     );
-
+    
     await databases.createDocument(
-        todoDatabase.$id,
-        todoCollection.$id,
-        sdk.ID.unique(),
+        todoDatabase.$id, 
+        todoCollection.$id, 
+        sdk.ID.unique(),
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:172]:
```suggestion
        todoDatabase.$id,
        todoCollection.$id,
        sdk.ID.unique(),
```

File: src/routes/docs/quick-starts/node/+page.markdoc [hunk 18]
```diff
@@ -184,7 +184,7 @@ Run the functions to by calling `runAllTasks();`.
 ```js
 async function getTodos() {
     var todos = await databases.listDocuments(
-        todoDatabase.$id,
+        todoDatabase.$id,
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/node/+page.markdoc:187]:
```suggestion
        todoDatabase.$id,
```

File: src/routes/docs/tutorials/android/+page.ts [hunk 19]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/android/step-1');
+	throw redirect(303, '/docs/tutorials/android/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/android/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/android/step-1');
```

File: src/routes/docs/tutorials/apple/+page.ts [hunk 20]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/apple/step-1');
+	throw redirect(303, '/docs/tutorials/apple/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/apple/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/apple/step-1');
```

File: src/routes/docs/tutorials/flutter/+page.ts [hunk 21]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/flutter/step-1');
+	throw redirect(303, '/docs/tutorials/flutter/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/flutter/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/flutter/step-1');
```

File: src/routes/docs/tutorials/nuxt/+page.ts [hunk 22]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-    redirect(303, '/docs/tutorials/nuxt/step-1');
+    throw redirect(303, '/docs/tutorials/nuxt/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/nuxt/+page.ts:5]:
```suggestion
    redirect(303, '/docs/tutorials/nuxt/step-1');
```

File: src/routes/docs/tutorials/nuxt/step-5/+page.markdoc [hunk 23]
```diff
@@ -27,7 +27,7 @@ const user = useUserSession();
       <!-- Email and logout button if logged in user -->
       <div
         class="main-header-end u-margin-inline-end-16"
-        v-if="user.current.value"
+        v-if="user.current.value"
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/nuxt/step-5/+page.markdoc:30]:
```suggestion
        v-if="user.current.value"
```

File: src/routes/docs/tutorials/nuxt/step-7/+page.markdoc [hunk 24]
```diff
@@ -103,7 +103,7 @@ const user = useUserSession();
     <IdeasList />
   </div>
 </template>
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/nuxt/step-7/+page.markdoc:106]:
```suggestion

```

File: src/routes/docs/tutorials/react/+page.ts [hunk 25]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/react/step-1');
+	throw redirect(303, '/docs/tutorials/react/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/react/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/react/step-1');
```

File: src/routes/docs/tutorials/refine/+page.ts [hunk 26]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-    redirect(303, '/docs/tutorials/refine/step-1');
+    throw redirect(303, '/docs/tutorials/refine/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/refine/+page.ts:5]:
```suggestion
    redirect(303, '/docs/tutorials/refine/step-1');
```

File: src/routes/docs/tutorials/subscriptions-with-stripe/+page.ts [hunk 27]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/subscriptions-with-stripe/step-1');
+	throw redirect(303, '/docs/tutorials/subscriptions-with-stripe/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/subscriptions-with-stripe/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/subscriptions-with-stripe/step-1');
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/+page.ts [hunk 28]
```diff
@@ -1,5 +1,6 @@
 import { redirect } from '@sveltejs/kit';
+import type { PageLoad } from './$types';
 
-export function load() {
-    redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
-}
+export const load: PageLoad = async () => {
+	throw redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/+page.ts:5]:
```suggestion
    throw redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/step-1/+page.markdoc [hunk 29]
```diff
@@ -12,7 +12,7 @@ category: Auth
 Appwrite takes away your stress of building and maintaining a backend. Appwrite helps you implement authentication, databases, file storage, and respond to real-time events with **secure** APIs out of the box. 
 If you're a Svelte developer, examples in this guide shows you how Appwrite can help you add authentication to Svelte apps faster.
 
-# Before you start {% #before-you-start %}
+## Before you start
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/step-1/+page.markdoc:15]:
```suggestion
# Before you start {% #before-you-start %}
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/step-2/+page.markdoc [hunk 30]
```diff
@@ -36,14 +36,14 @@ create-svelte version 3.2.0
 └  Your project is ready!
 ```
 
-After the prompt is finished, you can head over to the newly created project.
+After the prompt is finished, you can head over to the newly create project.
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/step-2/+page.markdoc:39]:
```suggestion
After the prompt is finished, you can head over to the newly created project.
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/step-2/+page.markdoc [hunk 31]
```diff
@@ -36,14 +36,14 @@ create-svelte version 3.2.0
...
 
 ```sh
 cd my-svelte-project
 npm install
 ```
 
-# Adding Appwrite to your Svelte app {% #adding-appwrite-to-your-svelte-app %}
+## Adding Appwrite to Your Svelte App
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/step-2/+page.markdoc:46]:
```suggestion
# Adding Appwrite to your Svelte app {% #adding-appwrite-to-your-svelte-app %}
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/step-3/+page.markdoc [hunk 32]
```diff
@@ -38,7 +38,7 @@ export const appwrite = {
 
 For example, your `.env` might look something similar to this.
 
-```env
+```text
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/step-3/+page.markdoc:41]:
```suggestion
```env
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/step-7/+page.markdoc [hunk 33]
```diff
@@ -1,11 +1,11 @@
...
 layout: tutorial
 title: All set
-description: Add authentication to a SvelteKit project using Appwrite.
+description: Add Authentication to a SvelteKit project using Appwrite.
 step: 7
 ---
 If you want to see these authentication concepts applied in a more robust manner, you can see them in action in this [demo app](https://github.com/appwrite/getting-started-projects/tree/main/svelte/auth).
 
-# Other authentication methods {% #other-authentication-methods %}
+## Other authentication methods
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/step-7/+page.markdoc:9]:
```suggestion
# Other authentication methods {% #other-authentication-methods %}
```

File: src/routes/docs/tutorials/sveltekit/+page.ts [hunk 34]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/sveltekit/step-1');
+	throw redirect(303, '/docs/tutorials/sveltekit/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/sveltekit/step-1');
```

File: src/routes/docs/tutorials/vue/+page.ts [hunk 35]
```diff
@@ -2,5 +2,5 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async () => {
-	redirect(303, '/docs/tutorials/vue/step-1');
+	throw redirect(303, '/docs/tutorials/vue/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/vue/+page.ts:5]:
```suggestion
	redirect(303, '/docs/tutorials/vue/step-1');
```

File: src/routes/docs/products/auth/accounts/+page.markdoc [hunk 36]
```diff
@@ -5,26 +5,24 @@ description: Unlock advanced user management - Appwrite's Account API for seamle
...
+[phone (SMS)](/docs/products/auth/phone-sms),
+[Anonymous](/docs/products/auth/anonymous),
+[magic URL](/docs/products/auth/magic-url), and
+[OAuth 2](/docs/products/auth/oauth2)
+authentication.
 
 # Preferences {% #preferences %}
 
-You can store user preferences on a user's account using Appwrite's [Update preferences](/docs/references/cloud/client-web/account#updatePrefs) endpoint. You can store preferences such as theme, notification settings, or preferred language so they can be synced across multiple devices.
+You can store user preferences on a user's account using Appwrite's [Update Preferences](/docs/references/cloud/client-web/account#updatePrefs) endpoint. You can store preferences such as theme, notification settings, or preferred language so they can be synced across multiple devices.
```
[the remark this answers is not in the document]
Comment (gewenyu99) [reply src/routes/docs/products/auth/accounts/+page.markdoc:25]: We actually use sentence case.

File: src/routes/docs/products/auth/anonymous/+page.markdoc [hunk 37]
```diff
@@ -10,7 +10,7 @@ Anonymous sessions allow you to implement **guest** users. Guest users let you s
 
 # Create anonymous session {% #createSession %}
 
-Create an anonymous session with [Create Anonymous Session](/docs/references/cloud/client-web/account#createAnonymousSession) endpoint.
+Create an anonymous session with [Create Anonymous Session](/docs/references/cloud/client-web/account#createAnonymousSession) route.
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/anonymous/+page.markdoc:13]:
```suggestion
Create an anonymous session with [Create Anonymous Session](/docs/references/cloud/client-web/account#createAnonymousSession) method.
```

File: src/routes/docs/products/auth/oauth2/+page.markdoc [hunk 38]
```diff
@@ -1,33 +1,34 @@
...
 OAuth authentication allows users to log in using accounts from other popular services. This can be convenient for users because they can start using your app without creating a new account. It can also be more secure, because the user has one less password that could become vulnerable.
 
-When using OAuth to authenticate, the authentication request is initiated from the client application. The user is then redirected to an OAuth2 provider to complete the authentication step, and finally, the user is redirected back to your application.
+When using OAuth to authenticate, the authentication request is initiated from the client application. The user is then redirected to an OAuth 2 provider to complete the authentication step, and finally, the user is redirected back to the client application.
 
-# Configure OAuth2 login {% #configure %}
+# Configure OAuth 2 login {% #configure %}
 
-Before using OAuth2 login, you need to enable and configure an OAuth2 provider.
+Before using OAuth 2 login, you need to enable and configure an OAuth 2 login provider.
```
[the remark this answers is not in the document]
Comment (gewenyu99) [reply src/routes/docs/products/auth/oauth2/+page.markdoc:13]: We call it OAuth2 in the console. If we change, we change everywhere.

File: src/routes/docs/products/auth/oauth2/+page.markdoc [hunk 39]
```diff
@@ -148,22 +128,22 @@ let account = Account(client)
...
 
-After authenticating a user through their OAuth2 provider, you can fetch their profile information such as their avatar image or name. To do this you can use the access token from the OAuth2 provider and make API calls to the provider.
+After authenticating a user through their OAuth 2 provider, you can fetch their profile information such as their avatar image or name. To do this you can use the access token from the OAuth 2 provider and make API calls to the provider.
 
-After creating an OAuth2 session, you can fetch the session to get information about the provider.
+After creating an OAuth 2 session, you can fetch the session to get information about the provider.
 
 {% multicode %}
 ```js
+// Web SDK
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/oauth2/+page.markdoc:146]:
```suggestion
```

File: src/routes/docs/products/auth/oauth2/+page.markdoc [hunk 40]
```diff
@@ -179,6 +159,7 @@ console.log(session.providerAccessToken);
 ```
 
 ```dart
+// Flutter SDK
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/oauth2/+page.markdoc:162]:
```suggestion
```

File: src/routes/docs/products/auth/oauth2/+page.markdoc [hunk 41]
```diff
@@ -198,6 +179,7 @@ print(session.providerAccessToken);
 ```
 
 ```kotlin
+// Android SDK
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/oauth2/+page.markdoc:182]:
```suggestion
```

File: src/routes/docs/products/auth/oauth2/+page.markdoc [hunk 42]
```diff
@@ -218,6 +200,7 @@ print(session.providerAccessToken);
 ```
 
 ```swift
+// Apple SDK
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/oauth2/+page.markdoc:203]:
```suggestion
```

File: src/routes/docs/products/auth/quick-start/+page.markdoc [hunk 43]
```diff
@@ -89,7 +89,7 @@ mutation {
 
 # Login {% #login %}
 
-After you've created your account, users can be logged in using the [Create Email Password Session](/docs/references/cloud/server-nodejs/account#createEmailPasswordSession) endpoint.
+After you've created your account, users can be logged in using the [Create Email Session](/docs/references/cloud/client-web/account#createEmailSession) route.
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/quick-start/+page.markdoc:92]:
```suggestion
After you've created your account, users can be logged in using the [Create Email Session](/docs/references/cloud/client-web/account#createEmailSession) route.
```
```suggestion
After you've created your account, users can be logged in using the [Create Email Session](/docs/references/cloud/client-web/account#createEmailSession) method.
```

[the remark this answers is not in the document]
Comment (gewenyu99) [reply src/routes/docs/products/auth/quick-start/+page.markdoc]: We will use method.

**Review by eldadfux — APPROVED** [review]

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 44]
```diff
@@ -169,14 +168,13 @@ async function seedDatabase(): Promise<void> {
         ID.unique(),
         testTodo1
     );
-
+
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:171]:
```suggestion

```

File: src/routes/docs/quick-starts/deno/+page.markdoc [hunk 45]
```diff
@@ -64,7 +64,7 @@ import { Client, ... other imports } from "https://deno.land/x/appwrite/mod.ts";
 {% /section %}
 {% section #step-4 step=4 title="Import Appwrite" %}
 
-Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
```
Comment (TorstenDittmann) [inline src/routes/docs/quick-starts/deno/+page.markdoc:67]:
```suggestion
Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
```

File: src/routes/docs/tutorials/sveltekit-csr-auth/+page.ts [hunk 46]
```diff
@@ -1,5 +1,6 @@
 import { redirect } from '@sveltejs/kit';
+import type { PageLoad } from './$types';
 
-export function load() {
-    redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
-}
+export const load: PageLoad = async () => {
+    throw redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
```
Comment (TorstenDittmann) [inline src/routes/docs/tutorials/sveltekit-csr-auth/+page.ts:5]:
```suggestion
    redirect(303, '/docs/tutorials/sveltekit-csr-auth/step-1');
```

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**